student/models.py

```diff
@@ -1,3 +1,30 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Gender(models.TextChoices):
+    MALE = "M", "Male"
+    FEMALE = "F", "Female"
+    UNSPECIFIED = "U", "Unspecified"
+
+
+class EducationalInstitution(models.Model):
+    institution_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.institution_name
+
+
+class Student(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    email = models.EmailField(max_length=255)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=1, choices=Gender.choices, default=Gender.UNSPECIFIED)
+    phone_number = models.CharField(max_length=10, blank=True, null=True)
```

Comment:
`len("+972-54-1234567") > 10`

Reply:
Also having both `blank=True` and `null=True` seldom makes sense.

Reply:
I'll change the max length to 15 and leave only blank as true